src/bin/cql-stress-cassandra-stress/java_generate/values/boolean.rs

```diff
@@ -0,0 +1,28 @@
+use scylla::frame::response::result::CqlValue;
+
+use crate::java_generate::distribution::Distribution;
+
+use super::{ValueGenerator, ValueGeneratorFactory};
+
+#[derive(Default)]
+pub struct Boolean;
+
+impl ValueGenerator for Boolean {
+    fn generate(
+        &mut self,
+        _identity_distribution: &mut dyn Distribution,
+        _size_distribution: &mut dyn Distribution,
+    ) -> CqlValue {
+        // For some reason original c-s retunrs:
+        // identity_distribution.next_i64() % 1 == 0 which is always true.
+        CqlValue::Boolean(true)
```

Comment:
returns

Reply:
Wow, no one fixed it in c-s code for 10 years...

Reply:
@Lorak-mmk WDYT?

Reply:
Is it that important to be bug-compatible with cql-stress? Imo we should do it the right way, I doubt anyone depends on this bug.

Reply:
I changed it so we generate a truly random `boolean` value. Removed the compat test cases for booleans from CI, since we are not compatible with c-s in this case.